bitnami/postgresql/templates/_helpers.tpl

```diff
@@ -33,7 +33,7 @@ Create the default FQDN for PostgreSQL read-only replicas headless service
 We truncate at 63 chars because of the DNS naming spec.
 */}}
 {{- define "postgresql.readReplica.svc.headless" -}}
-{{- printf "%s-hl" (include "postgresql.readReplica.fullname" .) | trunc 63 | trimSuffix "-" }}
+{{- printf "%s-hl" (include "postgresql.readReplicas.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Comment:
```suggestion
{{- printf "%s-hl" (include "postgresql.readReplica.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Reply:
Seems that it was correct before the change, the function is called `readReplica`